### ISSUE TITLE
Eksport areal og observasjoner for koder

### ISSUE THREAD

**Issue by bjornreppen** [Artsdatabanken/grunnkart-dataflyt#63]
Eksporter en json som inneholder:

 - kode
 - totalt kartlagt areal av typen i m² (inklusive barn)
- antall observerte unike arter

Blokkerer [ratatouille#570](https://github.com/Artsdatabanken/ratatouille/issues/570)

**Comment by runningbones** [comment]
Menes kun kartlagt (NA), eller omfattes også koder som har polygoner/areal fra andre kilder (eks. AO, MI, VV etc)?

**Comment by bjornreppen** [comment]
Ja, for alle koder uansett prefiks var det jeg hadde sett for meg?  Ordet kartlagt var antagelig litt uheldig å bruke.

**Comment by jarped** [comment]
Sql: https://github.com/Artsdatabanken/grunnkart-dataflyt/commit/87e54340f078e8c97627d3e131f3456ae7a09530

**Comment by jarped** [comment]
Jobben aggregerer bare ett nivå så langt.

**Comment by jarped** [comment]
https://test.artsdatabanken.no/data/json/codes/areasAndObservationsPerCode.json

**Comment by bjornreppen** [comment]
Ser ut som det er antall observasjoner som kommer ut  istedet for antall unike arter?

**Comment by bjornreppen** [comment]
Ser ut som kommuner mangler areal

**Comment by bjornreppen** [comment]
Ser ut som det også mangler areal på alle 1:5000 (koder med *-C-*)

**Comment by jarped** [comment]
Høres ut som vi trenger en ekstra runde for å hente alle på nederste nivå. Bør være lett å fikse.
